### ISSUE TITLE
Change how we expose the schema information in the docs

### ISSUE THREAD

**Issue by maruiz93** [kubearchive/kubearchive#523]
We currently have a [Database Schema page](https://kubearchive.github.io/kubearchive/main/reference/database.html) in our docs that we manually maintain to keep up with the changes on indexes, fields, PKs and FKs.

It is hard to keep up with the changes and the format of this page doesn't provide much value over the DDL files.

I propose use any automation tool for creating a diagram based on the schema, which is more visual and implies less manual and error prone work on our side.

On a separate issue we could investigate how we could end up having this task automated in our CI so nobody needs to bother on manually extracting the diagram image.

It could also be useful to have a link to the actual DDL definitions.

*Acceptance criteria*:
* Replace the content of the Database Schema section in our docs for links to the actual DDL definitions and an image with the visual diagram.

**Comment by rh-hemartin** [comment]
We need to take into account that the documentation is versioned and the schema could change from version to version.

**Comment by maruiz93** [comment]
Yes, that applies for the links to the DDL definitions (linked to the file of the proper tag) and I guess that for the link to the image too
